modules/virt-about-vm-templates.adoc

```diff
@@ -15,6 +15,8 @@ The *Templates* tab displays four types of virtual machine templates:
 * *Red Hat Provided* templates have limited support from Red Hat.
 * *User Provided* templates are *Red Hat Provided* templates that were cloned and created by users.
 
+You can use the filters in the template *Catalog* to sort the templates by attributes such as boot source availability, operating system, and workload.
+
 [NOTE]
 ====
 In the Templates tab, you cannot edit or delete Red Hat Supported or Red Hat Provided templates. You can only edit or delete custom virtual machine templates that were created by users.
```

Comment:
Not related to your changes, but `s/Templates/*Templates*` and the "Red Hat Supported/Provided" phrases should also be bolded if they're UI elements